h/api/db.py

```diff
@@ -140,15 +22,14 @@ def bind_engine(engine, base=Base, should_create=False, should_drop=False):
 
 
 def includeme(config):
-    """Configure and possibly initialize Elasticsearch and its models."""
-    registry = config.registry
-    settings = registry.settings
+    """Configure Elasticsearch client."""
+    settings = config.registry.settings
+
+    if 'es.host' in settings:
```

Comment:
Then the underlying code will default to `http://localhost:9200/annotator`: https://github.com/openannotation/annotator-store/blob/45267b04047210d26ab39b40570e3dd28179a49d/annotator/elasticsearch.py#L28-L29